fotoapparat/src/main/java/io/fotoapparat/parameter/range/IntervalRange.java

```diff
@@ -0,0 +1,58 @@
+package io.fotoapparat.parameter.range;
+
+import android.support.annotation.NonNull;
+
+/**
+ * Implementation of {@link Range} that represents numeric interval.
+ *
+ * @param <T> type of numbers in that interval.
+ */
+public class IntervalRange<T extends Number & Comparable<T>> implements Range<T> {
```

Comment:
Make sense. What are you thoughts on creating some factory method for creating ```Range```? Now I see a lot of ```ContinuousInterval``` instance creation in code and I doubt that in future this class couldn't be replaced by another class for a reason.